### ISSUE TITLE
Point Tally sound continues playing forever if you close out of Daily Battle result too quicky

### ISSUE THREAD

**Issue by BusterBeachside** [RunnersRevival/revival_issues#56]
See video: https://youtu.be/NPvZ31gpgJQ

When viewing the result of a Daily Battle, closing out of the window before the point tally has completed will cause the sound to play infinitely. This can be corrected by opening the result window again and letting the animation play out fully. It persists in menus, but starting a run also removes the sound.

**Comment by FairPlay137** [comment]
A fix for this oversight has already been implemented in 2.3.0, pending release.

When I first implemented the reworked sound code for that window in a privately-distributed server feature test APK based on 2.0.3, I didn't test for this edge case for some reason. This code would make its way as-is (if my memory serves) into 2.1.0 and would remain unchanged and undiscovered until Daily Battles became a proper feature on our server.
